osgbuild/fetch_sources.py

```diff
@@ -81,6 +81,13 @@ def process_meta_url(line, destdir):
         sha1 = output.split()[0]
         if sha1 != git_hash:
             raise Error("Repository hash %s corresponding to tag %s does not match expected hash %s" % (sha1, tag, git_hash))
```

Comment:
Shouldn't we eliminate this check since `git_hash` doesn't necessarily have to match the default cloned state of the local repo?